nbconvert/exporters/exporter.py

```diff
@@ -97,6 +97,7 @@ class Exporter(LoggingConfigurable):
             "nbconvert.preprocessors.ExtractOutputPreprocessor",
             "nbconvert.preprocessors.ExtractAttachmentsPreprocessor",
             "nbconvert.preprocessors.ClearMetadataPreprocessor",
+            "nbconvert.preprocessors.NumberedHeadingsPreprocessor",
```

Comment:
Just added the processor here, otherwise it was not running `python -m nbconvert --to=html --NumberedHeadingsPreprocessor.enabled=true notebook.ipynb` (even though the option to enable it appeared in `python -m nbconvert --help-all `